### PR TITLE
bug bash fixes 

### PR DESCRIPTION
**Pull request by aidanliddiard**

### CODE REVIEW

**Review by tbyrd2237 — APPROVED** [review]
Looks good.

File: frontend/src/components/DeleteItineraryModal.tsx [hunk 1]
```diff
@@ -23,7 +24,7 @@ const DeleteModal: React.FC<DeleteModalProps> = ({
       console.error("Error deleting itinerary:", error);
     }
     const updatedItinerary = await getItinerary(vacationId);
-    setItinerary(updatedItinerary)
+    setItinerary(updatedItinerary);
```
Comment (lesleytomosada) [inline frontend/src/components/DeleteItineraryModal.tsx:27]: hahaha thank you for adding this. i'm bad at remembering to add ;s

File: frontend/src/views/LogIn.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function LogIn() {
                     name="email"
                     id="email"
                     className="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                    placeholder="name@company.com"
+                    placeholder="Ex. name@company.com"
```
Comment (lesleytomosada) [inline frontend/src/views/LogIn.tsx]: for consistency it should be Ex: instead of Ex.

File: frontend/src/views/SignUp.tsx [hunk 3]
```diff
@@ -72,13 +77,14 @@ export default function SignUp() {
...
                   Email
+                  <span className="text-red-500 text-small"> *</span>
                 </label>
                 <input
                   type="email"
                   name="email"
                   id="email"
                   className="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                  placeholder="name@company.com"
+                  placeholder="Ex. name@company.com"
```
Comment (lesleytomosada) [inline frontend/src/views/SignUp.tsx]: this one too

**Review by lesleytomosada — APPROVED** [review]
good work!

**Review by tbyrd2237 — APPROVED** [review]

**Pull request merged**